download.sh

```diff
@@ -28,17 +28,10 @@ popd
 
 
 install_fabric_manager () {
-    FABRIC_MANAGER_VERSION="${DRIVER_VERSION}"
-    if [[ "${DRIVER_VERSION}" == "535.161.08" ]]; then
-      FABRIC_MANAGER_VERSION="535.104.05"
-      # TODO Temporary as latest fabric manager version here is not the same as the driver version. https://developer.download.nvidia.com/compute/cuda/redist/fabricmanager/linux-x86_64/
-    fi
-
-    curl -fsSLO https://developer.download.nvidia.com/compute/cuda/redist/fabricmanager/linux-x86_64/fabricmanager-linux-x86_64-${FABRIC_MANAGER_VERSION}-archive.tar.xz
-
-    tar -xvf fabricmanager-linux-x86_64-${FABRIC_MANAGER_VERSION}-archive.tar.xz
-    mv fabricmanager-linux-x86_64-${FABRIC_MANAGER_VERSION}-archive /opt/gpu/fabricmanager-linux-x86_64-${FABRIC_MANAGER_VERSION}
-    mv /opt/gpu/fm_run_package_installer.sh /opt/gpu/fabricmanager-linux-x86_64-${FABRIC_MANAGER_VERSION}/sbin/fm_run_package_installer.sh
+    curl -fsSLO https://developer.download.nvidia.com/compute/nvidia-driver/redist/fabricmanager/linux-x86_64/fabricmanager-linux-x86_64-${DRIVER_VERSION}-archive.tar.xz
```

Comment:
Just to note: this one was shared by Henry. This has the tar.xz file.
 
Nvidia folks shared this for all components in Ubuntu 22.04 but it doesn't seem to have it in the same format though: https://developer.download.nvidia.com/compute/cuda/repos/ubuntu2204/x86_64/